### ISSUE TITLE
How to describe the service endpoints in k8s Python client?

### ISSUE THREAD

**Issue by rameshar16** [kubernetes-client/python#1712]
Hi All,
Is it possible to describe the service endpoints in the k8s Python client?

Thanks and Regards,
Ramesh

**Comment by showjason** [comment]
Yes, please refer to function "read_namespaced_endpoints" in [python](https://github.com/kubernetes-client/python)/[kubernetes](https://github.com/kubernetes-client/python/tree/master/kubernetes)/[client](https://github.com/kubernetes-client/python/tree/master/kubernetes/client)/[api](https://github.com/kubernetes-client/python/tree/master/kubernetes/client/api)/core_v1_api.py

**Comment by rameshar16** [comment]
Thank you @showjason.
